utils/ort/src/main/kotlin/Constants.kt

```diff
@@ -29,6 +29,16 @@ const val ORT_NAME = "ort"
  */
 const val ORT_FULL_NAME = "OSS Review Toolkit"
```

Comment:
out of scope: Should we somehow put these constants into a namespace? (either a `Constants` object, or a `constants` package)? The former could be nice for discoverability via IDE auto completion.

Reply:
> The former could be nice for discoverability via IDE auto completion.

So far I haven't missed that feature, but I'd also have no objections.